nuxeo-features/nuxeo-platform-publisher/nuxeo-platform-publisher-core/src/main/java/org/nuxeo/ecm/platform/publisher/impl/finder/AbstractRootSectionsFinder.java

```diff
@@ -100,12 +100,10 @@ public DocumentModelList getSectionRootsForWorkspace(DocumentModel currentDoc, b
         }
 
         if (unrestrictedDefaultSectionRoot.isEmpty() && addDefaultSectionRoots) {
-            if (unrestrictedDefaultSectionRoot == null || unrestrictedDefaultSectionRoot.isEmpty()) {
-                DocumentModelList defaultSectionRoots = getDefaultSectionRoots(session);
-                unrestrictedDefaultSectionRoot = new ArrayList<String>();
-                for (DocumentModel root : defaultSectionRoots) {
-                    unrestrictedDefaultSectionRoot.add(root.getPathAsString());
-                }
+            DocumentModelList defaultSectionRoots = getDefaultSectionRoots(session);
+            unrestrictedDefaultSectionRoot = new ArrayList<String>();
```

Comment:
You can remove String between angle brackets

Reply:
We have so many of those still, this will wait for a global cleanup.